cmd/analyzer/parse_args.go

```diff
@@ -18,6 +21,10 @@ type InArgs struct {
 	VPC                   *string
 	Debug                 *bool
 	Version               *bool
+	ConnectionDescription *string
+	Protocol              string
+	SrcMinPort            int64
+	SrcMaxPort            int64
 }
```

Comment:
Please add min and max port also for dst
Also, consider adding Q before these new properties or something else so that it will be clear these are relevant only in query mode